riscv/src/large_field/code_gen.rs

```diff
@@ -443,6 +443,31 @@ fn preamble(field: KnownField, runtime: &Runtime, with_bootloader: bool) -> Stri
         tmp3_col = 1 - XXIsZero
     }
 
+    // ================= ground field arithmetic =================
+
+    // Inverts the Goldilocks field value in memory pointed by register X,
+    // returning the low and high limbs in registers Y and Z, respectively.
+    //
+    // Unsolvable if X points to 0.
+    instr invert_gl X, Y
```

Comment:
should we always have the instruction though? for other things we require the coprocessor to be explicitly "requested".
I think it's fine like this, we should anyway add an optimization pass that removes unused instructions

Reply:
I think this is fine because it is a single instruction, not a whole new machine.

Reply:
that's still at least a couple extra columns if unused, but fine